ledger/acctdeltas.go

```diff
@@ -476,29 +476,13 @@ func (a *compactAccountDeltas) accountsLoadOld(tx trackerdb.TransactionScope) (e
 	}()
 	for _, idx := range a.misses {
 		addr := a.deltas[idx].address
-		ref, acctDataBuf, err := arw.LookupAccountDataByAddress(addr)
-		switch err {
-		case nil:
-			if len(acctDataBuf) > 0 {
-				persistedAcctData := &trackerdb.PersistedAccountData{Addr: addr, Ref: ref}
```

Comment:
there is still a difference. this code does not set `Round` but `LookupAccount` does set. I did not check if this is important or not, but I suggest to either check or clear it here

Reply:
The `Round` is the round of the db at the point the "old value" was read. 
The old code wasn't setting it because it wasn't querying it, since it didn't seem to care about what round the old value was from. `LookupAccount` reads the round, because it's part of the data structure being read, and it might not be used by this caller, but it is used by others. 

This can't possible break code, because if any code was attempting to get meaning from the `Round` field in this "old value" entries, they all had `Round(0)`, implying that all this old values were read at genesis, which is flat out wrong.

Reply:
a problem could be in a code that uses `Round=0` as some indication similarly to `rowid=0`. I'm pretty sure there is no such code, but have you checked? The PR description claimed the code is equivalent but it is not for this case and for the empty data as discussed in a thread below.

Reply:
Hello, isn't the account data hashed by the catchpoint tracker, so wouldn't populating it with a new nonzero round number change that hash?

Reply:
You are right Round might affect MT hash but we've checked all possible code paths and returning Round=0 not possible now.